wallet/res/values/strings-extra.xml

```diff
@@ -27,22 +27,33 @@
     <string name="dialog_confirm_fee">Network fee</string>
     <string name="dialog_confirm_total">Total</string>
 
-    <string name="quick_action_secure_now">Secure\nWallet Now</string>
-    <string name="quick_action_scan_to_pay">Scan to\nSend</string>
-    <string name="quick_action_pay_to_address">Send to\nAddress</string>
-    <string name="quick_action_buy_sell">Buy / Sell\nDash</string>
-    <string name="quick_action_import_key">Import\nPrivate Key</string>
-    <string name="quick_action_add_shortcut">Add\nShortcut</string>
-
-    <string name="lock_action_quick_receive">Quick\nReceive</string>
-    <string name="lock_action_login_with_pin">Login\nwith PIN</string>
-    <string name="lock_action_login_with_fingerprint">Login with\nFingerprint</string>
-    <string name="lock_action_scan_to_pay">Scan to\nSend</string>
+    <!-- Home Screen: Shortcut Button: 2 lines, 9 characters across -->
+    <string name="quick_action_secure_now">Secure Wallet Now</string>
+    <!-- Home Screen: Shortcut Button: 2 lines, 9 characters across -->
+    <string name="quick_action_scan_to_pay">Scan to Send</string>
+    <!-- Home Screen: Shortcut Button: 2 lines, 9 characters across -->
+    <string name="quick_action_pay_to_address">Send to Address</string>
+    <!-- Home Screen: Shortcut Button: 2 lines, 9 characters across -->
+    <string name="quick_action_buy_sell">Buy / Sell Dash</string>
+    <!-- Home Screen: Shortcut Button: 2 lines, 9 characters across -->
+    <string name="quick_action_import_key">Import Private Key</string>
+    <!-- Home Screen: Shortcut Button: 2 lines, 9 characters across -->
+    <string name="quick_action_add_shortcut">Add Shortcut</string>
+
+    <!-- Lock Screen: Button: 2 lines, 9 characters across -->
+    <string name="lock_action_quick_receive">Quick Receive</string>
+    <!-- Lock Screen: Button: 2 lines, 9 characters across -->
+    <string name="lock_action_login_with_pin">Login with PIN</string>
+    <!-- Lock Screen: Button: 2 lines, 9 characters across -->
+    <string name="lock_action_login_with_fingerprint">Login with Fingerprint</string>
+    <!-- Lock Screen: Button: 2 lines, 9 characters across -->
+    <string name="lock_action_scan_to_pay">Scan to Send</string>
+    <!-- Lock Screen: Text above the PIN Entry area -->
```

Comment:
In this section, the line breaks were removed.  Comments were added to indicate limited space.

Reply:
On small and large devices the text still looks the same without the line break in the text.  For example

Scan to
Pay